React/Views/ScrollView/RCTScrollView.m

```diff
@@ -231,10 +231,10 @@ - (void)setContentOffset:(CGPoint)contentOffset
 #endif // macOS]
   if (contentView && _centerContent && !CGSizeEqualToSize(contentView.frame.size, CGSizeZero)) {
     CGSize subviewSize = contentView.frame.size;
-#if TARGET_OS_OSX // [macOS
-    CGSize scrollViewSize = self.contentView.bounds.size;
-#else // [macOS
+#if !TARGET_OS_OSX // [macOS]
     CGSize scrollViewSize = self.bounds.size;
+#else // [macOS
+    CGSize scrollViewSize = self.contentView.bounds.size;
```

Comment:
These were the wrong way around, which made it confusing to understand.

Reply:
Thanks for the catch! I thought I found all the ones the wrong way around 😅